gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/jvmkill.c

```diff
@@ -17,9 +17,18 @@
 #include <stdio.h>
 #include <string.h>
 #include <unistd.h>
-
+#include <stdlib.h>
 #include <jvmti.h>
 
+void logMessage(char * message){
```

Comment:
Space before brace.

Instead of having to format the message by callers it will be much simpler to do it right here. You can use vfprintf to take care of variable argument list like:

```c
void logMessage(char *format, ...) {
   va_list args;
   va_start(args, format);
   ...
   vfprintf(logFile, format, args);
   va_end(args);
}
```

With this the callers can just call logMessage like fprintf passing format and all the arguments.

Reply:
Modified accordingly. other suggestions are also incorporated.